internal/network/command_peering_create.go

```diff
@@ -0,0 +1,212 @@
+package network
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *peeringCommand) newCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create <name>",
+		Short: "Create a peering.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.create,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Create an AWS VPC peering.",
+				Code: "confluent network peering create aws-peering --network n-abcde1 --cloud aws --aws-account 012345678901 --aws-vpc vpc-abcdef0123456789a --aws-routes 172.31.0.0/16,10.108.16.0/21",
+			},
+			examples.Example{
+				Text: "Create a GCP VPC peering.",
+				Code: "confluent network peering create gcp-peering --network n-abcde1 --cloud gcp --gcp-project temp-123456 --gcp-vpc-network customer-test-vpc-network --gcp-import-custom-routes",
+			},
+			examples.Example{
+				Text: "Create an Azure VNet peering.",
+				Code: "confluent network peering create azure-peering --network n-abcde1 --cloud azure --azure-tenant 1111tttt-1111-1111-1111-111111tttttt --azure-vnet /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/my-rg/providers/Microsoft.Network/virtualNetworks/my-vnet --customer-region centralus",
+			},
+		),
+	}
+
+	addNetworkFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddCloudFlag(cmd)
+	cmd.Flags().String("aws-account", "", "AWS account ID associated with the VPC that you are peering with Confluent Cloud network.")
+	cmd.Flags().String("gcp-project", "", "Google Cloud project ID associated with the VPC that you are peering with Confluent Cloud network.")
+	cmd.Flags().String("azure-tenant", "", "Azure Tenant ID in which your Azure Subscription exists.")
+	cmd.Flags().String("aws-vpc", "", "AWS VPC ID that you are peering with Confluent Cloud network.")
+	cmd.Flags().String("gcp-vpc-network", "", "Name of the Google Cloud VPC that you are peering with Confluent Cloud network.")
+	cmd.Flags().String("azure-vnet", "", "Azure Resource ID of the VNet that you are peering with Confluent Cloud.")
+	cmd.Flags().StringSlice("aws-routes", nil, "A comma-separated list of CIDR blocks of the AWS VPC that you are peering with Confluent Cloud network.")
+	cmd.Flags().Bool("gcp-import-custom-routes", false, "Enable customer route import for Google Cloud VPC Peering.")
+	cmd.Flags().String("customer-region", "", "Cloud region ID of the AWS VPC or Azure VNet that you are peering with Confluent Cloud network.")
```

Comment:
@kabraham13 Could you please review these flags and examples? 🙏

Reply:
Synced with @kabraham13 and @brianstrauch. We are condensing the flags as follows 
`cloud-account`: `aws-account`, `gcp-project`, and `azure-tenant`
`virtual-network`: `aws-vpc`, `gcp-vpc-network`, and `azure-vnet`